.travis.yml

```diff
@@ -6,17 +6,12 @@ install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/ma
 script: bash -ex ./.travis-docker.sh
 env:
   global:
+  - DISTRO="ubuntu"
   - PINS="httpaf-async:. httpaf-lwt:. httpaf:."
+  - PACKAGE="httpaf"
+  - TESTS=true
+  - POST_INSTALL_HOOK="opam install --with-test httpaf-async httpaf-lwt && opam exec -- make examples"
   matrix:
-  - PACKAGE="httpaf"       DISTRO="ubuntu-16.04"    OCAML_VERSION="4.07.1"
-  - PACKAGE="httpaf-async" DISTRO="ubuntu-16.04"    OCAML_VERSION="4.07.1"
-  - PACKAGE="httpaf-lwt"   DISTRO="ubuntu-16.04"    OCAML_VERSION="4.07.1"
-  - PACKAGE="httpaf"       DISTRO="ubuntu-16.04"    OCAML_VERSION="4.06.0"
-  - PACKAGE="httpaf-async" DISTRO="ubuntu-16.04"    OCAML_VERSION="4.06.0"
-  - PACKAGE="httpaf-lwt"   DISTRO="ubuntu-16.04"    OCAML_VERSION="4.06.0"
-  - PACKAGE="httpaf"       DISTRO="ubuntu-16.04"    OCAML_VERSION="4.04.2"
-  - PACKAGE="httpaf-async" DISTRO="ubuntu-16.04"    OCAML_VERSION="4.04.2"
-  - PACKAGE="httpaf-lwt"   DISTRO="ubuntu-16.04"    OCAML_VERSION="4.04.2"
-  - PACKAGE="httpaf"       DISTRO="debian-unstable" OCAML_VERSION="4.03.0"
-  - PACKAGE="httpaf-async" DISTRO="debian-unstable" OCAML_VERSION="4.03.0"
-  - PACKAGE="httpaf-lwt"   DISTRO="debian-unstable" OCAML_VERSION="4.03.0"
+  - OCAML_VERSION="4.07"
+  - OCAML_VERSION="4.06"
+  - OCAML_VERSION="4.05"
```

Comment:
This looks fine, but I'd recommend you also add `OCAML_VERSION=4.04` and `4.03` lines as well, since that's what the opam file advertises as supported versions.  There aren't any C bindings or anything in this, so the multiple distros won't matter much and will be tested in opam-repository when released.